setup.py

```diff
@@ -40,11 +43,14 @@
         "Natural Language :: English",
         "Operating System :: Unix",
         "Operating System :: MacOS :: MacOS X",
+        "Operating System :: Microsoft :: Windows :: Windows 10"
         "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
         "Topic :: Software Development :: Libraries",
         "Topic :: Software Development :: Version Control",
         "Topic :: Utilities",
     ],
+    scripts=['git-big-windows-setup'],
```

Comment:
Could we add this as a new command to the existing git-big commands so that it shows up in the help?